#!/usr/bin/env python
PACKAGE = "uav_ros_lib"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("noise_mv",     double_t,    0,     "Kalman Filter measured value noise",          1,    0,  500)
gen.add("noise_pos",    double_t,    0,     "Kalman Filter distance noise",                1,    0,   500)
gen.add("noise_vel",    double_t,    0,     "Kalman Filter distance rate noise",           1,    0,  500)

exit(gen.generate(PACKAGE, "uav_ros_lib", "ConstantVelocityLKFParameters"))